src/server.ts

```diff
@@ -11,6 +11,13 @@ const port = process.env.PORT || 8080;
 createVerifier();
 
 app.use('/api', api);
+
+api.use((req, res, next) => {
```

Comment:
I assume this is in relation to the 'cors' heading option in the frontend? This may not be sufficient, since stuff will not always be coming from localhost

Reply:
Yeup its the cors stuff. I figured this would have to be reconfigured for prod somehow, not sure what url we would be using though